rotary_jet_spinning/models/modelling.py

```diff
@@ -2,24 +2,24 @@
 # -*- coding: utf-8 -*-
 
 from math import *
```

Comment:
why do you import all math?
if you are using only a few math function you can import only these. e.g. if you use only sqrt() you can do:
`from math import sqrt`